src/Algolia.Search/Clients/SearchConfig.cs

```diff
@@ -88,6 +88,11 @@ public SearchConfig(string applicationId, string apiKey) : base(applicationId, a
             hosts.AddRange(commonHosts);
 
             DefaultHosts = hosts;
+
+            Compression = CompressionType.GZIP;
```

Comment:
If it's the default value, don't we want to change it for `NONE`?

Reply:
👍 Just updated it.